### PR TITLE
New UI components based on bootstrap

### PR DESCRIPTION
**Pull request by Enigmazflo**
1. Added user privilege indicator
2. Dynamic User account picture using initials
3. Switched everything to bootstrap component 
4. Made mobile friendly, scalable
5. Made a consistent design pattern

### CODE REVIEW

File: gui/control.js [hunk 1]
```diff
@@ -48,12 +48,17 @@ function render() {
...
     accountsContainer.appendChild(accountsDiv);
 
     const state = PSS.returnAccountList();
     const {accounts, currentUser} = state;
 
     logger.debug(JSON.stringify(accounts));
+    let exec = require('child_process').exec;
+    exec('NET SESSION', function(err,so,se) {
+        document.getElementById("admin-priv").innerHTML = se.length === 0 ? `<span class='text-primary'>Admin</span>` : `<span class='text-secondary'>User</span>`;
+    });
```
Comment (adsau59) [inline gui/control.js:61]: not readable move to some function and name it according to what it does

**Review by adam-saudagar — CHANGES_REQUESTED** [review]

File: gui/control.js [hunk 2]
```diff
@@ -26,10 +27,31 @@ function switchAccountHandler(nickname) {
...
+    let exec = require('child_process').exec;
+    exec('NET SESSION', function(err,so,se) {
+        let warning_user = `
+            <div class="alert alert-parsec-danger" role="alert">
+              <h4 class="alert-heading">Need Admin privilege!</h4>
+              <p>Parsec switcher needs admin access to work :(</p>
+              Please launch program with admin rights. 
+              <hr>
+              <p class="mb-0">Want to report bug ? Report it here: https://github.com/DefineX-Studios/parsec-account-switcher</p>
+            </div>`
```
Comment (adsau59) [inline gui/control.js:40]: better move html into index.html file and hide/show the dom here to not mix view and control

File: gui/control.js [hunk 3]
```diff
@@ -71,6 +95,16 @@ function render() {
...
+
+    const userPictures = document.querySelectorAll(".user-picture");
+    userPictures.forEach(function(picture) {
+        const username = picture.dataset.username;
+        picture.textContent = getInitials(username);
+    });
+
+    function getInitials(name) {
+        return name.split(" ").map(word => word[0]).join("").toUpperCase();
+    }
```
Comment (adsau59) [inline gui/control.js:107]: maybe move out of the render function, or in some util file as the function is generic

File: gui/control.js [hunk 4]
```diff
@@ -71,6 +95,16 @@ function render() {
             showToast("Error!", errorToMessage[error])
         });
     }
+
+    const userPictures = document.querySelectorAll(".user-picture");
+    userPictures.forEach(function(picture) {
+        const username = picture.dataset.username;
+        picture.textContent = getInitials(username);
+    });
```
Comment (adsau59) [inline gui/control.js:103]: instead of looping and changing things, why not do it at the time account cards are created

File: gui/control.js [hunk 5]
```diff
@@ -82,8 +116,18 @@ async function main() {
...
+    document.getElementById(`add-account-btn`).addEventListener('click', async function () {
+        const nickname = await showTextInputPopup("Enter the Nickname", "Add Account");
+        if (!nickname) {
+            showToast("Error!", "Enter a Valid name");
+            return;
+        }
+        const error = await PSS.addAccount(nickname);
+        if (!error) return;
+        showToast("Error!", errorToMessage[error])
+    });
```
Comment (adsau59) [inline gui/control.js:128]: why are we no longer using addButtonPressed() ?

**Review by adam-saudagar — CHANGES_REQUESTED** [review]

File: lib/account-handler.js [hunk 6]
```diff
@@ -18,23 +19,25 @@ async function addAccount(nickname){
...
+        logger.debug("Waiting for user to login")
+        const error2 = await switchAccountData(nickname)
+        if (error2) {
+            return error2
+        }
+        if (!await startProcess()) {
+            return error.START_ERROR
+        }
+        //todo logger
+        logger.debug("done")
```
Comment (adsau59) [inline lib/account-handler.js:37]: wtf were you thinking?

**Review by adam-saudagar — CHANGES_REQUESTED** [review]

**Review by adam-saudagar — APPROVED** [review]

**Pull request merged**